.github/workflows/pytest.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name:  docker logs
         run: docker logs ocean_aquarius_1 && docker logs ocean_provider_1 && docker logs ocean_provider2_1 && docker logs ocean_computetodata_1
         if: ${{ failure() }}
-      - name: Publish code coverage
-        uses: paambaati/codeclimate-action@v2.7.5
-        env:
-          CC_TEST_REPORTER_ID: ${{secrets.CC_TEST_REPORTER_ID}}
+      # - name: Publish code coverage
```

Comment:
Why is this commented?

Reply:
This is commented because PRs submitted from external forks do not have access to secrets like `CC_TEST_REPORTER_ID`, so publishing code coverage always fails.

Quoted from the PR description:

> **Warning**
> Revert the following commit before (or after) merging this PR. This commit skips the code coverage step in the CI which fails because CC_TEST_REPORTER_ID is not set in PRs by external contributors.
> - https://github.com/oceanprotocol/ocean.py/pull/860/commits/f26ef556d11009c01af2a507219ae3dae5d22c05 

There are two such issues. The other relates to `REMOTE_TEST_PRIVATE_KEY1`, see PR description for details.